app/controllers/application_controller.rb

```diff
@@ -125,8 +125,14 @@ def root_url_for_responsible_body(user)
   end
 
   def school_root_url_for(user)
-    if user.schools.one?
-      home_school_path(user.schools.first)
+    if user.schools.size == 1
+      if user.school.preorder_information&.school_will_order_devices? &&
+          user.school.preorder_information&.chromebook_info_still_needed? &&
+          !user.school.la_funded_provision?
```

Comment:
Maybe we could create a predicate method on the School instance with a name that describes the intent of the combination of these three predicate method calls?